client/src/components/NavBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default class NavBar extends React.Component {
+    handleLogout = ev => {
+        localStorage.removeItem('user-token');
+        this.props.history.push('/signin');
+    };
+    render(){
+        const token = localStorage.getItem('user-token');
+        return (
+            <nav className="nav-bar">
+                {token 
+                    ? <button className="logout" onClick={this.handleLogout}>Sign Out</button> 
+                    : <NavLink to="/signin" style={this.props.location.pathname === '/signup' ? { fontWeight: 'bold' } : null}>Sign In</NavLink>}
```

Comment:
Some of this should probably be broken up on multiple lines for the sake of readability:
```javascript
// ...
: (
  <NavLink
    to="/signin"
    style={
      this.props.location.pathname === '/signup' 
        ? { fontWeight: 'bold' } 
        : null
    }
  >
    Sign In
  </NavLink>
)
```